source/connection-string/tests/invalid-uris.yml

```diff
@@ -258,4 +258,13 @@ tests:
         auth: ~
         options: ~
 
-    
+    -
+        description: "mongodb+rawsrv without service name"
+        uri: "mongodb+rawsrv://tcp.test7.test.mongodb.com"
+        valid: false
```

Comment:
I'm admittedly not familiar with this, but if the point of `+rawsrv` is to prevent drivers from appending `_mongodb._tcp.` on their own, is it possible to have a valid test case where we manually add `_mongodb._tcp.` with `+rawsrv`?

Or is that specific to how DNS is configured and unrelated to the host name in the connection string here?

Reply:
I did add the `rawsrv-uri` test [here](https://github.com/mongodb/specifications/pull/1063/files#diff-1f7e89e066c8273e3140ac34a2e732f6ab7a2b4d5e267be662624cd936aa1a0cR1-R9) that uses `mongodb+rawsrv://_mongodb._tcp.test3.test.build.10gen.cc/` as a URI and ensures that hosts are correctly found through SRV. Does that count as a valid test case where `_mongodb._tcp.` is already included with `+rawsrv`?

Reply:
Oh, I missed that when I wrote the above comment. That's the kind of test I was expecting. Thanks!